### PR TITLE
Check `txpool_status` after every Load Test

### PR DESCRIPTION
**Pull request by bane**

### CODE REVIEW

**Review by goran-ethernal — APPROVED** [review]

File: .github/workflows/load-test.yml [hunk 1]
```diff
@@ -117,6 +117,12 @@ on: # yamllint disable-line rule:truthy
       max_gas_utilization:
         description: Maximum Gas Utilization
         value: ${{ jobs.load_test.outputs.max_gas_utilization }}
+      txpool_pending:
+        description: Number of transactions in pending
```
Comment (Stefan-Ethernal) [inline .github/workflows/load-test.yml]: Nitpick:
```suggestion
        description: Pending Transactions Count
```

File: .github/workflows/load-test.yml [hunk 2]
```diff
@@ -117,6 +117,12 @@ on: # yamllint disable-line rule:truthy
...
+        value: ${{ jobs.txpool_status.outputs.txpool_pending }}
+      txpool_queued:
+        description: Number of transactions in queued
```
Comment (Stefan-Ethernal) [inline .github/workflows/load-test.yml]: Nitpick:
```suggestion
        description: Queued Transactions Count
```

File: .github/workflows/notification-load-test.yml [hunk 3]
```diff
@@ -75,6 +75,14 @@ on: # yamllint disable-line rule:truthy
         description: Maximum Gas Utilization
         type: string
         required: true
+      txpool_pending:
+        description: Number of transactions in pending
```
Comment (Stefan-Ethernal) [inline .github/workflows/notification-load-test.yml]:
```suggestion
        description: Pending Transactions Count
```

File: .github/workflows/notification-load-test.yml [hunk 4]
```diff
@@ -75,6 +75,14 @@ on: # yamllint disable-line rule:truthy
...
+        type: string
+        required: true
+      txpool_queued:
+        description: Number of transactions in queued
```
Comment (Stefan-Ethernal) [inline .github/workflows/notification-load-test.yml]:
```suggestion
        description: Queued Transactions Count
```

**Review by Stefan-Ethernal — APPROVED** [review]

**Pull request merged**